dotnet-desktop-guide/net/winforms/whats-new/net80.md

```diff
@@ -0,0 +1,95 @@
+
+## Button commands
+
+Button commands were in preview with .NET 7, and is now fully enabled in .NET 8. Similar to WPF, the instance of an object that implements the <xref:System.Windows.Input.ICommand> interface can be assigned to the button's <xref:System.Windows.Forms.ButtonBase.Command%2A> property. When the button is clicked, the command is invoked.
+
+An optional parameter can be provided when the command is invoked, by the specifying a value for the button's <xref:System.Windows.Forms.ButtonBase.CommandParameter%2A> property.
+
+The `Command` and `CommandParameter` properties are set in the designer through the **Properties** window, under **(DataBindings)**, as illustrated by the following image.
+
+:::image type="content" source="media/net80/commands.png" alt-text="The Visual Studio properties window highlighting a Windows Forms' Button's Command and CommandParameter properties.":::
+
+Buttons also listen to the <xref:System.Windows.Input.ICommand.CanExecuteChanged%2A?displayProperty=nameWithType> event, which causes the control to query the <xref:System.Windows.Input.ICommand.CanExecute%2A?displayProperty=nameWithType> method. When that method returns `true`, the control is enabled; the control is disabled when `false`is returned.
+
+## Visual Studio DPI improvements
+
+Visual Studio 2022 17.8 Introduces DPI-unwaware designer tabs. Previously, the Windows Designer tab in Visual Studio ran at the DPI of Visual Studio. This causes problems when you're designing a DPI-unaware Windows Forms app. Now you can ensure that the designer runs at the same scale as you want the app to run, either DPI-aware or not. Before this feature was introduced, you had to run Visual Studio in DPI-unaware mode, which made Visual Studio itself blurry when scaling was applied in Windows. Now you can leave Visual Studio alone and let the designer run DPI-unaware.
+
+You can force the DPI-unaware designer by adding the `<ApplicationHighDpiMode>` project setting and setting the value to `true`.
+
+:::code language="xml" source="./snippets/net80/csharp/snippets.csproj" range="3-11" highlight="7":::
+
+> [!IMPORTANT]
+> After changing this setting, you must unload and reload your project to get Visual Studio to respect it.s
+
+## High DPI improvements
+
+High DPI rendering with <xref:System.Windows.Forms.HighDpiMode.PerMonitorV2> has been improved:
+
+- Correctly scale nested controls. For example, a button that's in a panel, which is placed on a tab page.
+- Scale <xref:System.Windows.Forms.Form.MaximumSize?displayProperty=nameWithType> and <xref:System.Windows.Forms.Form.MinimumSize?displayProperty=nameWithType> properties based on the current monitor DPI settings.
+
+  Starting with .NET 8, this feature is enabled by default and you need to opt out of it to revert to the previous behavior.
```

Comment:
We ended up leaving this as opt in, due to some instability with the designer code gen.

Reply:
@merriemcgaw , not this one. This is `ON `by default if users target to .NET 8.0. Are you referring to new `AnchorLayout`?